.build/deps.mk

```diff
@@ -18,6 +18,7 @@ deps: libdeps
 	$(ECHO_V)go install ./vendor/github.com/sectioneight/md-to-godoc
 	@$(call label,Installing interfacer...)
 	$(ECHO_V)go install ./vendor/github.com/mvdan/interfacer/cmd/interfacer
+	$(ECHO_V)richgo testfilter 2>/dev/null || ($(call label,Installing richgo) && go get github.com/sectioneight/richgo)
```

Comment:
this didn't work out of the box for me. i ran make and saw are issues with overalls. as a result, i manually upgraded overalls. i then saw that i was missing richgo so i manually installed it. my output is also not as pretty.

![screenshot 2017-02-02 13 58 35](https://cloud.githubusercontent.com/assets/899731/22570326/b01e7c48-e94f-11e6-9815-64422a6a619d.png)